kok-api/src/main/java/com/kok/kokapi/station/adapter/in/dto/response/RecommendedStationResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.kok.kokapi.station.adapter.in.dto.response;
+
+import com.kok.kokcore.station.domain.entity.Route;
+import com.kok.kokcore.station.domain.entity.Station;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public record RecommendedStationResponse(
+
+        List<String> routes,
+        Station station
+) {
+    public static RecommendedStationResponse of(Station station, List<Route> routes) {
+        return new RecommendedStationResponse(
+                routes.stream().map(Route::getName).collect(Collectors.toList()),
```

Comment:
```suggestion
                routes.stream().map(Route::getName).toList(),
```

아주 사소 ㅎㅎ

참고: https://stackoverflow.com/questions/65969919/differences-of-java-16s-stream-tolist-and-stream-collectcollectors-tolist